src/adapters/cloudflare/mod.rs

```diff
@@ -280,57 +299,59 @@ impl CloudflareClient {
-            "parameters": {
-                "datasets": ["cloudflare-workers"],
-                "filters": [
-                    {
-                        "key": "$metadata.service",
-                        "operation": "eq",
-                        "type": "string",
-                        "value": worker_name
-                    },
-                    {
-                        "key": "$workers.scriptVersion.id",
-                        "operation": "eq",
-                        "type": "string",
-                        "value": worker_version_id
-                    },
-                    {
-                        "key": "$workers.event.response.status",
-                        "operation": "gte",
-                        "type": "number",
-                        "value": status_code_range_start
-                    },
-                    {
-                        "key": "$workers.event.response.status",
-                        "operation": "lte",
-                        "type": "number",
-                        "value": status_code_range_end
-                    }
-                ],
-                "calculations": [
-                    {
-                        "key": "$workers.event.response.status",
-                        "operator": "count",
-                        "keyType": "number",
-                        "alias": "sum"
-                    }
-                ]
-            },
-            "timeframe": {
-                "to": to_timestamp,
-                "from": from_timestamp
-            }
+          "view": "calculations",
+          "queryId": "worker-calculation",
+          "parameters": {
+            "datasets": [
+              "cloudflare-workers"
+            ],
+            "filters": [
+              {
+                "key": "$metadata.service",
+                "operation": "eq",
+                "type": "string",
+                "value": worker_name
+              },
+              {
+                "key": "$workers.scriptVersion.id",
+                "operation": "eq",
+                "type": "string",
+                "value": worker_version_id
+              },
+              {
+                "key": "$workers.event.response.status",
+                "operation": "gte",
+                "type": "number",
+                "value": status_code_range_start
+              },
+              {
+                "key": "$workers.event.response.status",
+                "operation": "lte",
+                "type": "number",
+                "value": status_code_range_end
+              }
+            ],
+            "calculations": [
+              {
+                "key": "$workers.event.response.status",
+                "operator": "count",
+                "keyType": "number",
+                "alias": "sum"
+              }
+            ]
+          },
+          "timeframe": {
+            "to": to_timestamp,
+            "from": from_timestamp
+          }
         });
```

Comment:
Big-ass data structure. I'd pull this sucker out into one or more functions instead of inlining it in an already-long function.